### DISCUSSION TITLE
简悦和 Vimium 快捷键冲突的解决方案

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#3699] [使用技巧]
> 原文来自 https://www.jianshu.com/p/dbbb66b8ac7e

1.  右键点击 vimium 图标，单击选项，在 Custom key mappings 处输入以下命令：

```
unmap r
map R reload
```

![image](https://user-images.githubusercontent.com/81074/162608533-f5d6930b-03a8-4707-bcba-58a04e8c8e00.png)

解释：以上命令将原本重新加载页面的快捷键由 `r ` 变换为 `R`，从而可以避免影响简悦在阅读模式下通过 `sr` 命令开启快捷键模式。

## 关联问题

- https://github.com/Kenshin/simpread/issues/1966
- https://github.com/Kenshin/simpread/issues/972
- https://github.com/Kenshin/simpread/issues/169
- https://github.com/Kenshin/simpread/issues/270

**Comment by egon-g** [comment]
可以尝试临时关闭vimium
https://github.com/gdh1995/vimium-c/issues/113#issuecomment-580055009

**Reply by Kenshin** [reply]
哈哈，谢谢提示  👍
